site_scons/commandline.scons

```diff
@@ -17,10 +17,21 @@ AddOption(
 AddOption(
     "--e2_target",
     dest="e2_target",
-    default="dbt-build-release-oled",
     type="string",
     nargs=1,
     action="store",
     metavar="TARGET",
-    help='Build target directory as in e2studio, prefaced with "dbt-" instead of "e2-"',
+    help='Legacy single-target e2 option; target prefaced with "dbt-" instead of "e2-"',
```

Comment:
If you want to nuke this in this PR, the build dockerfile uses `dbt` as an entrypoint so modifying the `run`  in the "Run build" step should be sufficient to keep the build working. Something like this:

```diff
diff --git a/.github/workflows/build.yml b/.github/workflows/build.yml
index 6532849..f370332 100644
--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -26,7 +26,7 @@ jobs:
             --user=$(id --user):$(id --group) \
             -v $(pwd):/src \
             ghcr.io/bobtwinkles/deluge-ci-images:main \
-            --e2_target=dbt-build-${{ matrix.profile }}-${{ matrix.hardware }}
+            dbt-build-${{ matrix.profile }}-${{ matrix.hardware }}
       - name: Publish built firmware (bin)
         uses: actions/upload-artifact@v3
         with:
```

If there's another reason to keep the option around we can do that too.

Reply:
No reason to keep it around, but for any workflows others might have developed around it. I think I'd rather wait just as a show that I'm not out to introduce something only to break it the next day ;). Working with backward compatibility in mind and avoiding breaking legacy stuff too soon is probably a good way to approach this in governance. Maybe in a couple weeks now that I've changed the text to indicate that it's legacy.